### PR TITLE
Feature/377 create campaigns module

### PR DESCRIPTION
**Pull request by orospakr**

### CODE REVIEW

File: Sources/UI/Notifications/Services/NotificationStore/NotificationStoreService.swift [hunk 1]
```diff
@@ -1,166 +1,166 @@
...
+//
+//        guard let eventPipeline = self.eventPipeline else {
+//            return
+//        }
+//
+//        let attributes: Attributes = ["notification": notification]
+//        let event = EventInfo(name: "Notification Marked Read", namespace: "rover", attributes: attributes)
+//        eventPipeline.addEvent(event)
+//    }
+//}
```
Comment (orospakr) [inline Sources/UI/Notifications/Services/NotificationStore/NotificationStoreService.swift:166]: This remains only just for temporary reference, I'll be replacing it next PR with Core Data.

Comment (seanrucker) [reply Sources/UI/Notifications/Services/NotificationStore/NotificationStoreService.swift:166]: 👍

File: Sources/UI/UIAssembler.swift [hunk 2]
```diff
@@ -17,21 +17,12 @@ public struct UIAssembler {
     public var isLifeCycleTrackingEnabled: Bool
     public var isVersionTrackingEnabled: Bool
     
-    public var appGroup: String?
-    public var influenceTime: Int
-    public var isInfluenceTrackingEnabled: Bool
-    public var maxNotifications: Int
-    
-    public init(associatedDomains: [String] = [], urlSchemes: [String] = [], sessionKeepAliveTime: Int = 30, isLifeCycleTrackingEnabled: Bool = true, isVersionTrackingEnabled: Bool = true, appGroup: String? = nil, isInfluenceTrackingEnabled: Bool = true, influenceTime: Int = 120, maxNotifications: Int = 200) {
+    public init(associatedDomains: [String] = [], urlSchemes: [String] = [], sessionKeepAliveTime: Int = 30, isLifeCycleTrackingEnabled: Bool = true, isVersionTrackingEnabled: Bool = true, appGroup: String? = nil, influenceTime: Int = 120) {
```
Comment (seanrucker) [inline Sources/UI/UIAssembler.swift]: Should `appGroup` and `influenceTime` still be in there?

File: Sources/Campaigns/Extensions/UI.UNNotificationResponse.swift [hunk 3]
```diff
@@ -10,14 +10,14 @@ import Foundation
 import UserNotifications
 
 extension UNNotificationResponse {
```
Comment (seanrucker) [inline Sources/Campaigns/Extensions/UI.UNNotificationResponse.swift]: Comment should be `RoverCampaigns` and the filename should be `Campaigns.OSLog` not `UI.OSLog`.

Comment (seanrucker) [reply Sources/Campaigns/Extensions/UI.UNNotificationResponse.swift]: Also looks like the target is wrong.

**Review by seanrucker — CHANGES_REQUESTED** [review]

File: Sources/Campaigns/Extensions/UI.UNNotificationResponse.swift (hunk 3 above)
Comment (seanrucker) [reply Sources/Campaigns/Extensions/UI.UNNotificationResponse.swift]: The comment in the header is wrong.

**Review by seanrucker — CHANGES_REQUESTED** [review]
The comment in the header of the Campaigns.UNNotificationResponse.swift file is wrong.

**Review by seanrucker — APPROVED** [review]

**Pull request merged**